extras/static_code_analysis.sh

```diff
@@ -82,6 +86,23 @@ if $PERFORM_PEP8; then
 fi
 print_msg "" ""
 
+# The following messages report on the command line arguments IGNORE_MSG_xxx which indicate whether
+# static code analysis error messages will cause the Travis CI build to fail or are ignored.
+if $RUNS_ON_TRAVIS; then
+  if $IGNORE_MSG_VERA; then
+    print_msg "MSGBLD1010: " "IGNORE_MSG_VERA is set. VERA++ messages will not cause the build to fail."
+  fi
+  if $IGNORE_MSG_CPPCHECK; then
+    print_msg "MSGBLD1020: " "IGNORE_MSG_CPPCHECK is set. CPPCHECK messages will not cause the build to fail."
+  fi
+  if $IGNORE_MSG_CLANG_FORMAT; then
+    print_msg "MSGBLD1030: " "IGNORE_MSG_CLANG_FORMAT is set. CLANG_FORMAT messages will not cause the build to fail."
+  fi
+  if $RUNS_ON_TRAVIS && $IGNORE_MSG_PEP8; then
```

Comment:
@heplesser yes and no. I added the 'if $RUNS_ON_TRAVES' for 3 reasons.
- clarity
- robustness
If the script runs locally, executed by 'check_code_style.sh', and someone changes its parametrization, for whatever reason, it will still behave reasonable. The ignore flags do not make sense outside Travis.
- testing
I used the 'check_code_style.sh' script for testing. Means, you can do the above without getting the second string, e.g.  'IGNORE_MSG_VERA is set. VERA++ .....'.